src/core/lib/transport/error_utils.cc

```diff
@@ -44,6 +44,8 @@ static grpc_error* recursively_find_error_with_field(grpc_error* error,
   return nullptr;
 }
 
+extern grpc_slice_refcount NoopRefcount;
```

Comment:
I would suggest defining this in slice_internal.h so that everyone has the definition if/when needed.

Reply:
Done.